lib/setupmanagers/qemuhck/qemuhck.rb

```diff
@@ -152,6 +152,10 @@ def append_vms_info(logs)
       end
     end
 
+    def clients_post_start_commands
+      @clients_vm.values.first.post_start_commands
```

Comment:
`Hash#values` will create an `Array` that contains all values and `Array#first` disposes them except the first one, which is inefficient.
Use `Hash#each_value`, which allows to get only the first one with `Enumerator#first`.

```suggestion
      @clients_vm.each_value.first.post_start_commands
```